src/Microsoft.TemplateEngine.Edge/Installers/NuGet/NuGetVersionHelper.cs

```diff
@@ -9,6 +9,8 @@ namespace Microsoft.TemplateEngine.Edge.Installers.NuGet
 {
     internal static class NuGetVersionHelper
     {
+        private static readonly FloatRange _unspecifiedVersion = new FloatRange(NuGetVersionFloatBehavior.Major);
```

Comment:
a comment what is `new FloatRange(NuGetVersionFloatBehavior.Major)` will be useful - latest release version